nav2_minimal_tb4_description/hooks/tb4_description.dsv.in

```diff
@@ -0,0 +1 @@
+prepend-non-duplicate;GZ_SIM_RESOURCE_PATH;share/
```

Comment:
Environment hooks are a nice and clean way to set environment variables when sourcing the workspace

Reply:
Yes, very nice!